tests/Http/HttpTestingFileFactoryTest.php

```diff
@@ -9,6 +9,10 @@ class HttpTestingFileFactoryTest extends TestCase
 {
     public function testImagePng()
     {
+        if (! function_exists('imagejpeg')) {
```

Comment:
Should test against `imagepng`
`if (! function_exists('imagepng')) {......`

Reply:
Yeh. You're right. I'll update.